test/parsers/file_test.rb

```diff
@@ -1,23 +1,27 @@
 require 'test_helper'
 
-class Ffprober::Parsers::FileTest < Minitest::Test
-  class FakeExec
-    attr_writer :json_output
-    def json_output(file_to_parse)
-      @json_output || "{}"
-    end
-  end
+module FFprober
+  module Parsers
+    class FileTest < Minitest::Test
+      class FakeExec
+        attr_writer :json_output
+        def json_output(_file_to_parse)
+          @json_output || '{}'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.